doc/source/whatsnew/v0.20.0.txt

```diff
@@ -20,7 +20,7 @@ Check the :ref:`API Changes <whatsnew_0200.api_breaking>` and :ref:`deprecations
 
 New features
 ~~~~~~~~~~~~
-
+- ``pd.to_datetime`` has a new parameter, ``origin``, to define an offset for ``DatetimeIndex`` (:issue:`11276`, :issue:`11745`)
```

Comment:
move to 0.19.0. pls add a sub-section with an example (you can use same one in doc-string). in the enhancements section.